internal/common/resourceutil/cpu/cpu.go

```diff
@@ -10,24 +10,38 @@ import (
 )
 
 const (
+	// USER is the key for user. Value set to 0
 	USER = iota
+	// NICE is the key for Nice, Value set to 1
 	NICE
+	// SYSTEM  is the key for system, Value set to 2
 	SYSTEM
+	// IDLE is the key for idle, Value set to 3
 	IDLE
+	// IOWAIT is the key for IO wait, Value set to 4
 	IOWAIT
+	// IRQ is the key for IRQ, Value set to 5
 	IRQ
+	// SOFTIRQ is the key for SOFTIRQ, Value set to 6
 	SOFTIRQ
+	// STEAL is the key for steal, Value set to 7
 	STEAL
+	// GEUST is the key for guest, Value set to 8
 	GEUST
+	// GESTNICE is the key for guest nice, Value set to 9
 	GESTNICE
 )
 
 const (
-	CPU_IDLE = iota
-	CPU_NONIDLE
-	CPU_TOTAL
+	// CPUIdle is the key idle cpu. value set to 0
+	CPUIdle = iota
+	//CPUNonidle is key for non idle cpu
```

Comment:
```suggestion
	// CPUNonidle is key for non idle cpu
```

@nitu-s-gupta One last checker, although this is a trivial change. 😄

Reply:
Done